tools/testing_utils/experiments/experiment_tests.csv

```diff
@@ -99,6 +99,13 @@ testing_archive.test_scanbound_negative::Slice .* scanbound times must be non-ne
 testing_archive.test_scanbound_not_increasing::Slice .* scanbound times must be increasing
 testing_archive.test_intt_longer_than_scanbound::Slice .* intt .*ms longer than scanbound time .*s
 testing_archive.test_intt_longer_than_scanbounds::Slice .* intt .*ms longer than one of the scanbound times
+testing_archive.test_tx_antenna_pattern_not_callable::Slice .* tx antenna pattern must be a function
```

Comment:
should this file (test_tx_antenna_pattern_not_callable.py) not be empty? (the test file appears empty). I assume just putting a global called tx_antenna_pattern into a normalscan-like experiment would do the trick

Reply:
It should not be empty. I will make an actual test for it.